### PR TITLE
New CBM lore snippets (leaflets)

### PR DESCRIPTION
**Pull request by nexusmrsep**
#### Summary
```SUMMARY: Content "Adds new lore snippets (leaflets) for recently added CBMs"```

#### Purpose of change
Many proper CBMs have lore teasers in form of advertising leaflets. Few newly added CBM deserve some too.

#### Describe the solution
Added some snippets.
*If I missed ay newly added CBM that deserves an advertising leaflet, let me know, and I'll add it here.*

#### Describe alternatives you've considered
N/A

#### Additional context
N/A

### CODE REVIEW

File: data/json/snippets.json [hunk 1]
```diff
@@ -79,6 +79,11 @@
     "category": "flier",
     "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of several bikini-clad female doctors removing the skin from man's face.  The patient is strapped to a chair but cheerfully gives two thumbs up.  The caption reads: \"Free Plastic Surgery While You Wait!\"" ]
   },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of several bikini-clad nurses giving medicine to a delighted patient, who sucks it with a neede pointing out of his finger.  Outside the window two shadow figures are doing the same with something less resembling medicine.  The caption reads: \"Why use old IV when you can have Intravenous Needle CBM!\"" ]
```
Comment (Leland) [inline data/json/snippets.json]: > several bikini-clad nurses giving medicine to a delighted patient, who sucks it with a neede pointing out of his finger

`needle`

Also, he sucks what, exactly? 

> Outside the window two shadow figures are doing the same with something less resembling medicine

Two shadow figures are... providing medicine to a patient? Sorry, this third sentence confuses me a bit

File: data/json/snippets.json [hunk 2]
```diff
@@ -89,6 +94,16 @@
...
   },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of a man carrying a somewhat confused horse on his shoulders.  He is flanked by two scantily clad nurses who hold bloody chainsaws and give approving thumbs up.  His bulging knees and elbows are stiched and seem to be the source of his ability.  The caption reads:  \"Brand new!  Horse power at hand!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of a man smiling while being relentlesly punched by a boxer.  He doesn't even sweat as punches have no effect on him.  Two scantily clad cheerleader nurses give approving thumbs up from the background.  The caption reads:  \"Bouncy as never before!  Absorb those shocks!\"" ]
```
Comment (Qrox) [inline data/json/snippets.json]: What is a cheerleader nurse?

**Comment by Xhuis** [comment]
A small correction on the Needletip - it doesn't involve holding it and then ingesting it. Instead, it's a net of tubes that links directly into your bloodstream, so it'd be closer to dipping your finger in the source of whatever you were injecting and draining it straight into your body.

File: data/json/snippets.json (hunk 2 above)
Comment (nexusmrsep) [reply data/json/snippets.json]: I'd post a picture but that would get me kicked. But ok, lest cut out the confusion and just leave nurses.

**Comment by Inglonias** [comment]
These ads are kind of creepy. Not just yours, but if Sybil's was a real place with these ads, I would not be terribly interested in going there.

**Comment by Marrim** [comment]
What CBM does the confused horse one advertise?
...also there's a presumed typo "Stiched" in that one.

**Pull request merged**